src/frontend/screens/ExportObservations.tsx

```diff
@@ -0,0 +1,254 @@
+import {NativeRootNavigationProps} from '../sharedTypes/navigation';
+import {DARK_GREY, LIGHT_GREY, WARNING_RED} from '../lib/styles';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import {useState} from 'react';
+import {useExportGeoJSON, useExportZipFile} from '@comapeo/core-react';
+import {useActiveProject} from '../contexts/ActiveProjectContext';
+import * as FileSystem from 'expo-file-system';
+import {useOpenShareDialog} from '../hooks/share';
+import {useLocaleState} from '../contexts/LocaleStoreContext';
+import {UIActivityIndicator} from 'react-native-indicators';
+import {useObservations} from '../hooks/server/observations';
+import {useTracks} from '../hooks/server/track';
+import * as Sentry from '@sentry/react-native';
+
+const m = defineMessages({
+  close: {
+    id: 'screens.ExportObservations.close',
+    defaultMessage: 'Close',
+  },
+  download: {
+    id: 'screens.ExportObservations.download',
+    defaultMessage: 'Download',
+  },
+  allObservations: {
+    id: 'screens.ExportObservations.allObservations',
+    defaultMessage: 'All Observations',
+  },
+  allObservationsDescription: {
+    id: 'screens.ExportObservations.allObservationsDescription',
+    defaultMessage: 'Text only as a GeoJson file',
+  },
+  allObservationsAndMedia: {
+    id: 'screens.ExportObservations.allObservationsAndMedia',
+    defaultMessage: 'All Observations with Media',
+  },
+  allObservationsAndMediaDescription: {
+    id: 'screens.ExportObservations.allObservationsAndMediaDescription',
+    defaultMessage: 'Images and audio in a .zip file',
+  },
+  tracks: {
+    id: 'screens.ExportObservations.tracks',
+    defaultMessage: 'Tracks',
+  },
+  tracksDescription: {
+    id: 'screens.ExportObservations.tracksDescription',
+    defaultMessage: 'Tracks only as a GeoJson file',
+  },
+  errorMessage: {
+    id: 'screens.ExportObservations.errorMessage',
+    defaultMessage: 'Choose an option',
+  },
+});
+
+const EXPORT_OPTIONS = [
+  {
+    type: 'Observation',
+    description: m.allObservationsDescription,
+    title: m.allObservations,
+  },
+  {
+    type: 'ObservationsWithMedia',
+    description: m.allObservationsAndMediaDescription,
+    title: m.allObservationsAndMedia,
+  },
+  {type: 'Tracks', title: m.tracks, description: m.tracksDescription},
+] as const;
+type Exports = (typeof EXPORT_OPTIONS)[number]['type'];
+
+export const ExportObservations = ({
+  navigation,
+}: NativeRootNavigationProps<'ExportObservations'>) => {
+  const {formatMessage} = useIntl();
+  const [typeToExport, setTypeToExport] = useState<null | Exports>(null);
+  const [showError, setShowError] = useState<boolean>(false);
+  const {projectId} = useActiveProject();
+  const exportNoMedia = useExportGeoJSON({projectId});
+  const exportWithMedia = useExportZipFile({projectId});
+  const [loading, setLoading] = useState(false);
+  const openShare = useOpenShareDialog();
+  const lang = useLocaleState(store => store.languageTag);
```

Comment:
this should be using `useAppLanguageTag()`

```suggestion
  const lang = useAppLanguageTag();
```

Reply:
@comapeo/schema

Reply:
https://github.com/digidem/comapeo-mobile/pull/1192/commits/894bdaf6ace8834e43073f4b07750472f8634be3

Reply:
https://github.com/digidem/comapeo-mobile/pull/1192/commits/2a23d0fbb65f1daa925f993ec5d6f9dacb560f94